mowgli_etl/pipeline/wdc/wdc_transformer.py

```diff
@@ -95,16 +51,19 @@ def transform(
         self.__product_type_classifier = wdc_product_type_classifier
 
         # Parse file
-        with open(wdc_clean_file_path) as data:
+        with open(wdc_jsonl_file_path) as data:
```

Comment:
If you have the WdcOffersCorpus class I suggested, you can also put some of this reading there. (.read() instead of sample). That way the jsonl_file_path is in one place, and encapsulated away from this class.